articles/design/using-auth0-with-multi-tenant-apps.md

```diff
@@ -9,50 +9,48 @@ contentType: concept
 useCase: strategize
 ---
 
-# Using Auth0 with Multi-Tenant Applications
+# Using Auth0 to Secure Your Multi-Tenant Applications
 
-In this article, we'll discuss (at a high-level) how Auth0 can help you manage users for your multi-tenant applications.
+This article provides a high-level overview of how Auth0 can help you to secure your multi-tenant applications.
 
-If you're planning to build a multi-tenant solution, please [contact our Sales or Professional Services](https://auth0.com/?contact=true) teams.
+[Multi-tenancy](https://en.wikipedia.org/wiki/Multitenancy) is when a single instance of software runs on a server that is accessible to multiple groups of users. In fact, Auth0 itself is a multi-tenant application; your collection of applications, settings, and connections is called a "tenant". Note that this document is **not about using multiple Auth0 tenant(s)**, but is about using Auth0 to secure your own multi-tenant application.

```

Comment:
## What is multi-tenancy

Multi-tenancy is when a single instance of software runs on a server that is accessible to multiple groups of users.

Auth0's Public Cloud deployment is an example of a multi-tenant application. Your applications, settings, and connections are a single tenant, which shares resources with other tenants in the Public Cloud.

Please note that this article is **not about using multiple Auth0 tenant(s)**. It is about using Auth0 to secure your own multi-tenant application.

---------------------------------------------

I don't know if bringing back this header might be problematic?

Reply:
?